### ISSUE TITLE
Expected 'package', found 'EOF' issue

### ISSUE THREAD

**Issue by justmumu** [imroc/req#83]
Hi, 

I am getting error (expected 'package', found 'EOF') while downloading v2 version.

```
$ go get github.com/imroc/req/v2
../../../../go/pkg/mod/github.com/imroc/req/v2@v2.0.2/req_test.go:1:1: expected 'package', found 'EOF'
```

**Comment by imroc** [comment]
thanks, it was a little mistake which accidentally introduced and has been fixed

**Comment by justmumu** [comment]
thank you too man for quick fix

**Comment by Dmaskus** [comment]
../src/github.com/imroc/req/textproto_reader.go:479:9: undefined: io.ReadAll
../src/github.com/imroc/req/transport.go:1597:17: tlsConn.HandshakeContext undefined (type *tls.Conn has no field or method HandshakeContext)
../src/github.com/imroc/req/transport.go:1653:16: tc.HandshakeContext undefined (type *tls.Conn has no field or method HandshakeContext)

**Comment by imroc** [comment]
> ../src/github.com/imroc/req/textproto_reader.go:479:9: undefined: io.ReadAll ../src/github.com/imroc/req/transport.go:1597:17: tlsConn.HandshakeContext undefined (type *tls.Conn has no field or method HandshakeContext) ../src/github.com/imroc/req/transport.go:1653:16: tc.HandshakeContext undefined (type *tls.Conn has no field or method HandshakeContext)

@Dmaskus  thanks for you feedback. I just found that req uncompatible with low golang version, and fixed it just now (v2.0.4)
